src/nrrd/nrrd.h

```diff
@@ -428,7 +428,9 @@ typedef struct NrrdIoState_t {
                                used to allocate a buffer for header */
   void *oldData;            /* ON READ: if non-NULL, pointer to space that
                                has already been allocated for oldDataSize */
-  size_t oldDataSize;       /* ON READ: size of mem pointed to by oldData */
+  size_t oldDataSize,       /* ON READ: size of mem pointed to by oldData */
+    chunckElementNumber;    /* if != 0, a subset will be read or written and it indicates the
```

Comment:
actually I have experienced that it is better a long int (for very large chunk). I'll change to long int

Reply:
size_t is defined so that it can represent the size of the largest possible object on the target platform, so size_t should work (all memory allocation related methods use size_t), unless element size may be smaller than 1 byte